src/components/UserProfile/index.js

```diff
@@ -26,9 +20,9 @@ export default class UserProfile extends Component {
 
   render() {
 
-    const { name, email, photo, tab, toggleTab } = this.props;
-    const { favorites } = this.state;
-    const mappedFavorites = this.state.favorites.map( school => <Favorites school={ school } key={ school.name } /> );
+    const { name, email, photo, tab, toggleTab, favorites } = this.props;
+    
+    const mappedFavorites = favorites.map( school => <Favorites school={ school } commuteInfo={ false } key={ school.school_id } /> );
```

Comment:
setting commuteInfo here to false, though i think it comes through as 'undefined' so may could change to null, to conditionally make the schoolData object in the fav button component